elixir/HelloWorld/mix.exs

```diff
@@ -10,7 +10,7 @@ defmodule HelloWorld.MixProject do
       deps: deps(),
       atomvm: [
         start: HelloWorld,
-        flash_offset: 0x210000
```

Comment:
This should be in a separate PR (it can be together with the above correction) we just want to keep the new additions in a different commit from the corrections to other examples. It makes the commit log cleaner and more useful.